apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-button/index.ts

```diff
@@ -1,102 +0,0 @@
-import { recordTracksEvent } from '@automattic/calypso-analytics';
-import { dispatch } from '@wordpress/data';
-import domReady from '@wordpress/dom-ready';
-import { __ } from '@wordpress/i18n';
-import { inIframe } from '../../../block-inserter-modifications/contextual-tips/utils';
-import { GUTENBOARDING_LAUNCH_FLOW } from '../constants';
-
-import 'a8c-fse-common-data-stores';
-import '@wordpress/editor';
```

Comment:
Is this removing the last use of `@wordpress/editor`? Can we remove that dependency from editing toolkit altogether?